Source/Rhetos.Configuration.Autofac/RhetosContainerBuilder.cs

```diff
@@ -20,48 +20,42 @@
 using Autofac;
 using Rhetos.Extensibility;
 using Rhetos.Logging;
+using System;
+using System.Collections.Generic;
 
 namespace Rhetos
 {
     /// <summary>
-    /// Container builder initialized with <see cref="InitializationContext"/>.
-    /// It makes the <see cref="InitializationContext"/> and <see cref="IPluginScanner"/> accessible during registration process and
+    /// Container builder initialized with <see cref="IConfigurationProvider"/>.
+    /// It makes the <see cref="ILogProvider"/> and <see cref="IPluginScanner"/> accessible during registration process and
     /// registers <see cref="IConfigurationProvider"/> to the container.
     /// </summary>
     public class RhetosContainerBuilder : ContainerBuilder
     {
         /// <summary>
-        /// Initializes a container with specified <see cref="InitializationContext"/>. 
-        /// Registers the <see cref="IConfigurationProvider"/> instance to newly created container.
+        /// Initializes a container with specified <see cref="IConfigurationProvider"/>. 
+        /// Registers <see cref="IConfigurationProvider"/> instance to newly created container.
+        /// The assembly list is used for plugins search when using the <see cref="ContainerBuilderPluginRegistration"/>.
         /// <see cref="ILogProvider"/> is not registered and is meant to be used during the lifetime of registration and container building process.
         /// <see cref="LegacyUtilities"/> will also be initialized with the given configuration.
         /// </summary>
-        public RhetosContainerBuilder(InitializationContext initializationContext)
+        public RhetosContainerBuilder(IConfigurationProvider configurationProvider, ILogProvider logProvider, Func<List<string>> findAssemblies)

```

Comment:
Consider changing `findAssemblies` to `IAssemblyListProvider` or similar. Then `LegacyUtilities.GetListAssembliesDelegate()` could be just new instance of `class LegacyAssemblyListProvider : IAssemblyListProvider`.